Modules/socketmodule.c

```diff
@@ -7686,8 +7686,29 @@ PyInit__socket(void)
     PyModule_AddIntMacro(m, CAN_SFF_MASK);
     PyModule_AddIntMacro(m, CAN_EFF_MASK);
     PyModule_AddIntMacro(m, CAN_ERR_MASK);
-#ifdef CAN_ISOTP
+#ifdef HAVE_LINUX_CAN_ISOTP_H
```

Comment:
Please leave ``CAN_ISOTP`` in the ``ifdef CAN_ISOTP`` guard. The constant is declared in ``linux/can.h``.

```suggestion
#ifdef CAN_ISOTP
    PyModule_AddIntMacro(m, CAN_ISOTP);
#endif
#ifdef HAVE_LINUX_CAN_ISOTP_H
    PyModule_AddIntMacro(m, SOL_CAN_ISOTP);
```